MakeWin.bat

```diff
@@ -0,0 +1,150 @@
+if '%1'=='simphony-ncad'  goto simphony-ncad
+if '%1'=='nfluid'         goto nfluid
+if '%1'=='all' (
+                          call %0 simphony
+                          call %0 simphony-ncad
+                          call %0 nfluid
+                          goto exit
+               )
+if '%1'=='clean'          goto clean
+if '%1'=='help'           goto help
+if '%1'=='pip'            goto pip
+if '%1'=='numpy'          goto numpy
+REM ----- DEVELOPMENT -------------------------
+if '%1'=='nfluid-dev'     goto nfluid-dev
+REM -------------------------------------------
+goto help
+
+
+:simphony
+echo ----- INSTALLING SIMPHONY -----
+REM Check if pip is installed
+python -c "import pip" 2>nul
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if numpy is installed
+python -c "import numpy" 2>nul
+if %ERRORLEVEL% neq 0  call %0 numpy
+
+REM Retrieve and install simphony library
+pip install --upgrade git+https://github.com/simphony/simphony-common.git@%SIMPHONYVERSION%#egg=simphony
+goto exit
+
+
+:simphony-ncad
+echo ----- INSTALLING SIMPHONY-NCAD -----
+REM Check if pip is installed
+python -c "import pip" 2>nul
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if cython is installed
+python -c "import cython" 2>nul
+if %ERRORLEVEL% neq 0  pip install cython
+REM Check if simphony is installed
+python -c "import simphony" 2>nul
+if %ERRORLEVEL% neq 0  call %0 simphony
+
+pip install --upgrade git+https://github.com/simphony/simphony-nCAD.git@%NCADVERSION%#egg=simphony_ncad
+goto exit
+
+
+:nfluid
+echo ----- INSTALLING NFLUID -----
+REM Check if pip is installed
+python -c "import pip" 2>nul
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if numpy is installed
+python -c "import numpy" 2>nul
+if %ERRORLEVEL% neq 0  call %0 numpy
+REM Check if simphony is installed
+python -c "import simphony" 2>nul
+if %ERRORLEVEL% neq 0  call %0 simphony
+
+pip install --upgrade git+https://github.com/simphony/nfluid.git@%NFLUIDVERSION%#egg=nfluid
+goto exit
+
+
+:nfluid-dev
+echo ----- INSTALLING NFLUID-DEV -----
+pip install --upgrade git+https://github.com/simphony/nfluid.git@nfluid-goyo-dev#egg=nfluid-goyo-dev
+goto exit
+
+
+:clean
+echo ----- REMOVING SIMPHONY, SIMPHONY-NCAD AND NFLUID -----
+for /d %%f in ("C:\Python27\Lib\site-packages\ncad_wrapper*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\nfluid*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\simncad*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\simphony*") do rd /s /q "%%~f"
+goto exit
+
+
+:pip
+REM Get pip package
+REM -- using urllib
+python -c "import urllib; urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py', 'C:\python27\Tools\get-pip.py')"
+REM -- using urllib2
+REM python -c "import urllib2; ifile=urllib2.urlopen('https://bootstrap.pypa.io/get-pip.py');ofile=open('C:/python27/Tools/pip.py','wb');ofile.write(ifile.read());ofile.close()"
+REM Install pip
+python C:\python27\Tools\get-pip.py
```

Comment:
It would also be helpful to echo "Please add C:\Python27\Scripts\ to your PATH environment" (something like that) to help the user?